extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DevServicesBuildTimeConfig.java

```diff
@@ -21,7 +21,7 @@ public interface DevServicesBuildTimeConfig {
     /**
      * The container image name to use, for container based DevServices providers.
      *
-     * If the provider is not container based (e.g. a H2 Database) then this has no effect.
+     * This has no effect if the provider is not container-based database, such as H2 or Derby.
```

Comment:
The previous paragraph still has "container based", so this is now inconsistent.

Reply:
Fixing! :)